src/utils/matchingEpisodes.ts

```diff
@@ -0,0 +1,21 @@
+import { IEpisode } from "./interface";
+export function isMatching(
```

Comment:
would be easier to locate the function if the file had the same name, also the return is an array and not a boolean as the name of the function is suggesting.